.github/workflows/ci-checks.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 #v4.4.0
         with:
-          node-version: "20"
+          node-version: "22.12.0"
```

Comment:
I'd prefer to define `NODE_VERSION` variable in `env:` section of each file so it is more easier to change later.